app/stories/components/Button.stories.tsx

```diff
@@ -1,10 +1,16 @@
 import { ComponentMeta, ComponentStory } from '@storybook/react'
 
-import ButtonComponent from '@components/Button'
+import ButtonComponent, { buttonStyleOptions } from '@components/Button'
 
 export default {
   title: 'Components',
   component: ButtonComponent,
+  argTypes: {
+    style: {
+      control: 'select',
+      options: buttonStyleOptions,
+    },
+  },
```

Comment:
Notes for reviewers: Display the possible button style options as a dropdown in the storybook controls.

Reply:
<img width="526" alt="Screen Shot 2022-10-28 at 12 37 31 AM" src="https://user-images.githubusercontent.com/67701/198531125-7f427d2e-d91b-48ab-bc9f-71450f3587e1.png">

Reply:
If you define the type as a union type rather than `string`, this will be done automatically

Reply:
Oooo thank you!